src/components/Checkout/OrderSummary.js

```diff
@@ -1,163 +1,179 @@
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`$ 00.00`}</Text>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`Tax:`}</Text>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`$ 00.00`}</Text>
-				</GridItem>
-				<GridItem colSpan={2}>
-					<Divider />
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Heading size='l'>Order total:</Heading>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Heading size='l'>{`$ 00.00`}</Heading>
-				</GridItem>
-				<GridItem colSpan={2}>
-					<Button
-						disabled={isLoading || !stripe || !elements}
-						size='md'
-						w='full'
-						bg={colors.primary}
-						color='white'
-						onClick={handleSubmit}
-					>
-						<span id='button-text'>
-							{isLoading ? (
-								<div className='spinner' id='spinner'></div>
-							) : (
-								'Place Order'
-							)}
-						</span>
-					</Button>
-				</GridItem>
-				<GridItem colSpan={2}>
-					{message && (
-						<div
-							style={{
-								color: 'rgb(105, 115, 134)',
-								fontSize: '16px',
-								lineHeight: '20px',
-								paddingTop: '12px',
-								textAlign: 'center',
-							}}
-						>
-							{message}
-						</div>
-					)}
-				</GridItem>
-			</SimpleGrid>
-		</VStack>
-	);
+  return (
+    <VStack
+      w='full'
+      p={10}
+      spacing={10}
+      alignItems='flex-start'
+      style={{
+        border: "2px solid #E2E8F0",
+        borderRadius: "25px",
+      }}
+    >
+      <Heading size='xl'>Order Summary</Heading>
+      <SimpleGrid columns={2} rowGap={6} columnGap={2} w='full'>
+        <GridItem colSpan={1}>
+          <Text>{`Items(${calculateTotalItemsInCart()}):`}</Text>
+        </GridItem>
+        <GridItem colSpan={1}>
+          <Text>{`$ 000.00`}</Text>
+        </GridItem>
+        <GridItem colSpan={1}>
+          <Text>{`Shipping and handling:`}</Text>
+        </GridItem>
+        <GridItem colSpan={1}>
+          <Text>{`$ 00.00`}</Text>
+        </GridItem>
+        <GridItem colSpan={1}>
+          <Text>{`Tax:`}</Text>
+        </GridItem>
+        <GridItem colSpan={1}>
+          <Text>{`$ 00.00`}</Text>
+        </GridItem>
+        <GridItem colSpan={2}>
+          <Divider />
+        </GridItem>
+        <GridItem colSpan={1}>
+          <Heading size='l'>Order total:</Heading>
+        </GridItem>
+        <GridItem colSpan={1}>
+          <Heading size='l'>{`$ 00.00`}</Heading>
+        </GridItem>
+        <GridItem colSpan={2}>
+          {checkout ? (
+            <Link to='/checkout'>
+              <Button size='md' w='full' bg={colors.primary} color='white'>
+                Checkout
+              </Button>
+            </Link>
+          ) : (
+            <Button
+              disabled={isLoading || !stripe || !elements}
+              size='md'
+              w='full'
+              bg={colors.primary}
+              color='white'
+              onClick={handleSubmit}
+            >
+              <span id='button-text'>
```

Comment:
This isn't the only button on the page; it's better to use a more descriptive name.